config/migrations-generator.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+return [
+    // Where the templates for the generators are stored.
+    // Use this if you have n't your config file in ./config/migrations-generator.php
+    'migration_template_path'      => __DIR__ . '/../stubs/migration.generate.stub',
+    // // Use this if your config file is in ./config/migrations-generator.php and you want use default stub
+    // 'migration_template_path'      => base_path() . '/vendor/kitloong/laravel-migrations-generator/stubs/migration.generate.stub',
```

Comment:
We need a better way to implement this. Please revert to the old config.
The filename `migrations-generator.php` is okay.